components/org.wso2.carbon.appmgt.hostobjects/src/main/java/org/wso2/carbon/appmgt/hostobjects/APIProviderHostObject.java

```diff
@@ -426,6 +434,104 @@ public static NativeArray jsFunction_getBusinessOwners(Context cx, Scriptable th
         return myn;
     }
 
+    /**
+     * Retrieve business owner.
+     * @param cx      Rhino context
+     * @param thisObj Scriptable object
+     * @param args    Passing arguments
+     * @param funObj  Function object
+     * @return shared policy partials
+     * @throws org.wso2.carbon.appmgt.api.AppManagementException
+     */
+    public static NativeObject jsFunction_getBusinessOwner(Context cx, Scriptable thisObj, Object[] args,
+                                                           Function funObj) throws AppManagementException {
+        if (args == null || args.length != 1) {
+            handleException("Invalid number of input parameters.");
+        }
+        if (args[0] == null || args[0] == "null") {
+            handleException("Error while reading business owner. Owner id is null");
+        }
+        NativeObject row = new NativeObject();
```

Comment:
Fixed in https://github.com/wso2/carbon-appmgt/pull/539/commits/4642888b8ad6934dcda162142607b8549b0728e5